### DISCUSSION TITLE
Issue while using "SQLAnywhere17" Fluent nhibernate

### DISCUSSION THREAD

**Discussion by sravangontla** [nhibernate/fluent-nhibernate#516] [Q&A]
Hi Team,
  I am unable to perform DML operations using Fluent nhibernate. 

**Code :** 

                var sessionFactory = CreateSessionFactory();
                using (var session = sessionFactory.OpenSession())
                {
                    using (var transaction = session.BeginTransaction())
                    {
                        var newObject = new model{  };
                        session.SaveAsync(newObject);

                        transaction.Commit();
                    }
                }

**Error Details :**
  Error raised while calling transaction.commit() statement. And respective error is "System.IO.FileNotFoundException: 'Could not load file or assembly 'System.Data.Entity, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'. The system cannot find the file specified.' "

Env:
  netcoreapp3.1
  FluentNHibernate =>    Version="3.1.0"  

Thanks,
Sravan G

**Comment by hazzik** [comment]
Please provide the session factory configuration code.

**Comment by hazzik** [comment]
Please use `SybaseSQLAnywhereDriver` with [iAnywhere.Data.SQLAnywhere.NETCore](https://www.nuget.org/packages/iAnywhere.Data.SQLAnywhere.NETCore/) package or `OdbcDriver` with `System.Data.Odbc` package. The `SapSQLAnywhere17Driver` is .NET 4.5 only and does not support .NET Core.
